src/qt/choice.cpp

```diff
@@ -148,6 +149,10 @@ int wxChoice::DoInsertItems(const wxArrayStringsAdapter & items,
 int wxChoice::DoInsertOneItem(const wxString& item, unsigned int pos)
 {
     m_qtComboBox->insertItem(pos, wxQtConvertString(item));
+
+    if (m_style == wxCB_SORT)
```

Comment:
```suggestion
    if ( IsSorted() )
```
(look at its implementation in `wx/choice.h` to see why/how does it work)